apps/console/src/features/identity-providers/components/forms/authenticators/sms-otp-authenticator-form.tsx

```diff
@@ -349,22 +422,33 @@ export const SMSOTPAuthenticatorForm: FunctionComponent<SMSOTPAuthenticatorFormP
             validate={ validateForm }
         >
 
-            <Message
-                type={ "info" }
-                content={
-                    ( <Trans>
-                        <span>
-                            { t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +
-                                ".smsOTP.forTestingOnlyNotice.firstLine") }
-                        </span>
-                        <Divider hidden fitted/>
-                        <span>
-                            <Icon name="info circle" style={ { visibility: "hidden" } }/>
-                            { t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +
-                                ".smsOTP.forTestingOnlyNotice.secondLine") }
-                        </span>
-                    </Trans> ) }
-                width={ 13 }
+            {/*<Message*/}
```

Comment:
Lets avoid commenting if these will not be immediately brought in.

Reply:
resolved in https://github.com/wso2/identity-apps/pull/3699/commits/89a5b7ce383f598b61ffaab02e48b0a23c2bf27a